kernel-laws/src/main/scala/cats/kernel/laws/PartialOrderLaws.scala

```diff
@@ -0,0 +1,55 @@
+package cats.kernel.laws
+
+import cats.kernel.PartialOrder
+
+trait PartialOrderLaws[A] extends EqLaws[A] {
+  override implicit def E: PartialOrder[A]
+
+  def reflexitivity(x: A): IsEq[Boolean] =
+    E.gteqv(x, x) <-> true
+
+  def antisymmetry(x: A, y: A): IsEq[Boolean] =
+    (!(E.lteqv(x, y) && E.lteqv(y, x)) || E.eqv(x, y)) <-> true
+
+  def transitivity(x: A, y: A, z: A): IsEq[Boolean] =
+    (!(E.lteqv(x, y) && E.lteqv(y, z)) || E.lteqv(x, z)) <-> true
+
+  def gteqv(x: A, y: A): IsEq[Boolean] =
+    E.lteqv(x, y) <-> E.gteqv(y, x)
+
+  def lt(x: A, y: A): IsEq[Boolean] =
+    E.lt(x, y) <-> (E.lteqv(x, y) && E.neqv(x, y))
+
+  def gt(x: A, y: A): IsEq[Boolean] =
+    E.lt(x, y) <-> E.gt(y, x)
+
+  def partialCompare(x: A, y: A): IsEq[Boolean] = {
+    val c = E.partialCompare(x, y)
+    (((c < 0) == E.lt(x, y)) && ((c == 0) == E.eqv(x, y)) && ((c > 0) == E.gt(x, y))) <-> true
+  }
+
+  def pmin(x: A, y: A): IsEq[Option[A]] = {
+    val c = E.partialCompare(x, y)
+    val m = E.pmin(x, y)
+    if (c < 0) m <-> Option(x)
+    else if (c == 0) Option(x) <-> Option(y)
```

Comment:
hmm, this is a bit tricky, by using `IsEq[Option[A]]` we lost the test of `pmin` when the two are equal. I see two of options here,
1.  one change it to `IsEq[Boolean]` just like the old test
2.  change it to `IsEq[(Option[A], Option[A])]`, then you have to do `(m, m) <-> (Option(y), Option(y))`, `(m, m) <-> (Option(x), Option(y))`, etc. you can create a nested helper method to make the duplicated case  more readable
```scala
def isEqualTo(a: Option[A]) =  (m, m) <-> (a, a)
```
It seems to me that 1 is good enough and easier.